frontend/src/routes/Infrastructure/Clusters/ClusterPools/CreateClusterPool/CreateClusterPool.tsx

```diff
@@ -230,16 +230,6 @@ export function CreateClusterPool() {
                     control.validation.required = mustJoinClusterSet ?? false
                 }
                 break
-            case 'infrastructure':
```

Comment:
@chenz4027  I think we still need this though. This is not related to the reviewinfo.

Reply:
@fxiang1 I think this is vestigial code from when the infrastructure was selected as the first step of the wizard. Can you demonstrate when this code would run and have an effect?

Reply:
Yep you're right, we can go ahead and remove it. Thanks!